azure-pipelines.yml

```diff
@@ -23,13 +23,14 @@ jobs:
     - bash: |
         echo "Check Title of Pull Request: #$(System.PullRequest.PullRequestNumber)"
         title=$(curl https://api.github.com/repos/$(Build.Repository.Name)/pulls/$(System.PullRequest.PullRequestNumber) | jq -r '.title')
+        echo $title
+        body=$(curl https://api.github.com/repos/$(Build.Repository.Name)/pulls/$(System.PullRequest.PullRequestNumber) | jq -r '.body')
+        echo $body
         if [ "$(System.PullRequest.TargetBranch)" != "release" ] && echo $title | grep -iqF hotfix:; then
           echo "Hotfix PR should target release branch."
           exit 1
         fi
-        [[ $title =~ ^(\[.*\]|\{.*\}).* ]] && exit 0
-        echo "Pull Request title should follow https://aka.ms/submitAzPR"
-        exit 1
+        python /home/vsts/work/1/s/scripts/ci/check_pull_request.py "$title" "$body"
```

Comment:
We can use relative path, such as

https://github.com/Azure/azure-cli/blob/d85ce1fd1db6976a0ab0892e92980c2817cb8093/azure-pipelines.yml#L934